R/get-user-teams.R

```diff
@@ -0,0 +1,25 @@
+#' Get teams a user belongs to
+#'
+#' Looks up the teams a user belongs to. By default, it looks for teams of the
+#' current logged in user. You must be logged in to Synapse to use this
+#' function.
+#'
+#' @param user Synapser user object (e.g. output from
+#'   [synapser::synGetUserProfile()])
+#' @return Character vector of team IDs the user belongs to
+#' @examples
+#' \dontrun{
+#' get_user_teams()
+#' get_user_teams("karawoo")
+#' }
+get_user_teams <- function(user) {
+  user_teams <- synapser::synRestGET(
```

Comment:
Does synapser have the paginated rest get convenience function?

Reply:
Not sure which one you mean?

Reply:
The python client has a hidden function _GET_paginated which does the pagination of rest calls such that we don't have to set the limit really high or deal with getting all the results of a rest call.  Was just wondering if synapser had this hidden function.

Reply:
Ahh I didn't know about that. No, synapser doesn't have any of the hidden functions from the python client 😞

Reply:
There's been some jira discussion about this but no conclusions as far as I know https://sagebionetworks.jira.com/browse/SYNPY-996